spec-0007/index.md

```diff
@@ -24,11 +24,19 @@ We recommend:
 
 We suggest implementing these principles by:
 
-- deprecating the use of `numpy.random.seed` to control the random state,
+- deprecating user control of downstream library behavior with `numpy.random.seed`,
 - deprecating the use of `random_state`/`seed` arguments in favor of a consistent `rng` argument, and
 - using `numpy.random.default_rng` to validate the `rng` argument and instantiate a `Generator`.
 
-Note that `numpy.random.default_rng` does not accept instances of `RandomState`, so use of `RandomState` to control the seed is effectively deprecated, too.
+Note that `numpy.random.default_rng` does not accept instances of `RandomState`, so user control of library behavior with instances of `RandomState` is effectively deprecated, too.
```

Comment:
Again, not clear what the "user control of library behavior" distinction is  about; I'll go and look at Ralf's review comments that necessitated this change.

Reply:
I'm trying to make it explicit that it is an nteraction between users and libraries that is no longer allowed. 

I don't think we can just leave the old wording here and add the next change because the previous wording:

> use of `RandomState` to control the seed is effectively deprecated

seems contradicted by the new:

> neither `np.random.seed` nor `np.random.RandomState` themselves are deprecated

After all, what can you do with `np.random.seed` besides control the seed?

It means something more specific: code inside the library may no longer be controlled by a use of `np.random.seed` outside the library.

Reply:
```suggestion
Note that `numpy.random.default_rng` does not accept instances of `RandomState`, so use of `RandomState` to control the seed is effectively deprecated, too.
```